fighthealthinsurance/urls.py

```diff
@@ -14,31 +14,110 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 
-from typing import List, Union
+from typing import List, Union, Callable, Dict
 
 from django.urls import URLPattern, URLResolver
 from django.contrib import admin
-from django.http import HttpRequest, HttpResponseBase
+from django.http import HttpRequest, HttpResponseBase, HttpResponse, HttpResponseForbidden
 from django.contrib.admin.views.decorators import staff_member_required
 from django.contrib.staticfiles.urls import staticfiles_urlpatterns
 from django.urls import include, path, re_path
 from django.views.decorators.cache import cache_control, cache_page
 from django.views.decorators.csrf import csrf_exempt
 from django.views.generic.base import RedirectView
 from django.contrib.staticfiles.storage import staticfiles_storage
+from django.core.exceptions import PermissionDenied
+import re
+import time
 
 from fighthealthinsurance import views
 from fighthealthinsurance import fax_views
 from fighthealthinsurance import staff_views
 from django.views.decorators.debug import sensitive_post_parameters
 import os
-
+from django.conf import settings
 
 def trigger_error(request: HttpRequest) -> HttpResponseBase:
     raise Exception("Test error")
 
 
+# Add security middleware
```

Comment:
This does not belong in urls.py
Also this seems like there might be a library to do this rather than making our own?